roles/common/tasks/main.yml

```diff
@@ -23,3 +23,7 @@
 - name: assemble the authorized keys file
   assemble: dest=/root/.ssh/authorized_keys mode=0600 src=/tmp/pubkeys
   sudo: yes
+
+- name: install rackspace monitoring agent
+  script: monitoring.sh {{ rackspace_username }} {{ rackspace_api_key }}
```

Comment:
There's probably some Ansible-fu to make this only run the initialization script if the `rackspace-monitoring-agent` is not an available service, but this will certainly work!